src/snakemake/runtime_dependencies.py

```
@@ -0,0 +1,270 @@
+                )
+                # extract the requested packages with their determined versions from the
+                # uv output
+                posterior_packages = set(
+                    package
+                    for package in parse_uv_pip_dry_run_output(res.stderr)
+                    if package.name in self._packages[package_type]
+                )
+                if posterior_packages:
+                    # explicitly install the determined versions of the requested
+                    # packages under the prefix
+                    sp.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)